### DISCUSSION TITLE
How to add stop words when building BM25 index?

### DISCUSSION THREAD

**Discussion by RulinShao** [castorini/pyserini#1727] [Q&A] [answered]
Hello! Thanks for the great package! I'm wondering how I could pass stop words to pyserini when building a BM25 index? The default is not using any stop words, but I want to accelerate the construction and inference process by removing stop words. Thanks a lot!

**Comment by lintool** [comment, marked as answer]
Start here:
https://github.com/castorini/pyserini/blob/master/docs/usage-index.md#building-a-bm25-index-direct-java-implementation

The pyserini indexer is just a wrapper around the anserini indexer, and there's a `-stopwords` option to specify your own stopwords:
https://github.com/castorini/anserini/blob/master/src/main/java/io/anserini/index/IndexCollection.java#L144

Hope this helps!

**Reply by RulinShao** [reply]
Thank you!!! This resolves my question well!
